src/components/CourseOutlineStatus/displayMessages.jsx

```diff
@@ -0,0 +1,11 @@
+import { defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+  loadingChecklistLabel: {
+    id: 'loadingChecklistLabel',
+    defaultMessage: 'Loading',
```

Comment:
In this case, I wonder if this should be a little more explicit. Maybe "loading checklist score" or something?